src/internal_modules/roc_core/target_openssl/roc_core/secure_random.h

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
We can have only one secure_random.h (outside of target dirs) and two secure_random.cpp (inside target dirs)